Scripts/Impact_scripts/Finance.py

```diff
@@ -18,17 +18,36 @@
 import pandas as pd
 
 import sys
-sys.path.insert(0, '/***YOUR LOCAL FILE PATH***/CLOVER 4.0/Scripts/Conversion scripts')
+sys.path.insert(0, './Scripts/Conversion_scripts')
```

Comment:
In terms of `.`, the `os` module provides a good feature for this:
```
>>> os.getcwd()
'/home/bewinche'
```
This will automatically return the current working directory as an absolute directory. Should you want to use the `.` syntax, which you don't have to here, you can use `os.path.relpath()` function.